src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,33 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        Book createBook = new Book();
+        createBook.setTitle("King Arthur");
+        createBook.setPrice(BigDecimal.valueOf(355));
+
+        Book updateBook = new Book();
+        updateBook.setId(2L);
```

Comment:
id is set by DB

Reply:
I specified the id so that it was clear in the method which book in the table to change. Is it necessary to do it in a different way?